client/app/bundles/course/assessment/submission/components/FileInput.jsx

```diff
@@ -127,6 +153,19 @@ class FileInput extends Component {
             </Card>
           )}
         </Dropzone>
+        <Prompt
```

Comment:
![image](https://github.com/Coursemology/coursemology2/assets/42570513/3f1e0fdc-81f3-47fc-a986-ad17b21e5531)

"Cancel" does not suit the prompt here. The file upload has not been done so nothing to cancel. Should use "Close" or "Ok" instead.

ref: https://dribbble.com/tags/error_prompt

Reply:
done